components/vf-blockquote/vf-blockquote.njk

```diff
@@ -1,3 +1,20 @@
-<blockquote{% if id %} id="{{-id-}}"{% endif %} class="vf-blockquote{%- if override_class %} | {{override_class}}{% endif -%}">
-{{- html | safe if html else text -}}
```

Comment:
could we add a fallback to use `html` or `text` if `blockquote_text` is not set? (stops it from being a breaking change)

Reply:
I'll take a look

Reply:
it's a tongue twister but
```
{% if (blockquote_text) or (html) or (text) %}
    <p class="vf-blockquote__text">{{- (html) or (blockquote_text) | safe if (html) or (blockquote_text) else text -}}</p>
  {% endif %}
```

works for this

Reply:
@khawkins98 I'll let you resolve this and I'll merge when approved